core/shared/src/main/scala/com/lynbrookrobotics/potassium/units/spacialValues.scala

```diff
@@ -0,0 +1,58 @@
+package com.lynbrookrobotics.potassium.units
+
+import squants.space.{Area, Feet, Length}
+
+class Point(override val x: Length,
+            override val y: Length,
+            override val z: Length) extends Value3D(x, y, z) {
+  def this(value3D: Value3D[Length]) = this(value3D.x, value3D.y, value3D.z)
+  def this(x: Length, y: Length) = this(x, y, Feet(0))
+  def distanceTo(other: Point): Length = Segment(this, other).length
```

Comment:
Unfortunately I cannot because multiplication is not defined for all possible types of value3D. I can do it here, because multiplication is defined for Length